geoengine/workflow.py

```diff
@@ -7,11 +7,12 @@
 from __future__ import annotations
 
```

Comment:
In the notebook, could you output the structure of the array?

cf. https://corteva.github.io/rioxarray/html/examples/reproject.html where you have coordinates and variables.

Reply:
3e3b8f41d05f5080deef0bdb57bdc5f110448b51

Reply:
The output is

```
<xarray.Dataset>
Dimensions:      (x: 1024, y: 1024, time: 3, band: 2)
Coordinates:
  * x            (x) float64 -255.8 -255.2 -254.8 -254.2 ... 254.8 255.2 255.8
  * y            (y) float64 255.8 255.2 254.8 254.2 ... -254.8 -255.2 -255.8
  * time         (time) datetime64[ns] 2014-04-01T02:00:00 ... 2014-06-01T02:...
  * band         (band) int64 0 1
    spatial_ref  int64 0
Data variables:
    test         (time, band, y, x) float64 nan nan nan nan ... nan nan nan nan
```

1. I guess there should be so `test` variable
2. With rioxarray all bands are different data variables, but for us it is a dimension. Why is this so?

Reply:
1. yeah there should be no `test` variable, but I don't see it:
![image](https://github.com/geo-engine/geoengine-python/assets/10283892/5d5d9d97-5751-479f-87b7-1454d103ab5e)

2. you are right, in the rioxarray example it is modeled differently. I thought of the band as a dimension but it could also be a data variable. In this example here they model band as a coordinate so I guess there is no one right answer: https://corteva.github.io/rioxarray/html/examples/COG.html

Reply:
1. It is quite funny, because you committed it :smile: 
2. Interesting, maybe you could add the RGB example from what you've posted using our `ne2_raster_{color}` datasets?

Reply:
756c2d8d3e7d24dfcdf93c72f7a6bc2e9e6f623b